frontend/components/main.jsx

```diff
@@ -3,106 +3,119 @@ import { Link } from 'react-router-dom';
 import styled from 'styled-components';
 
 import { PEOPLE } from '../helpers';
-import { xsSpacing } from '../constants';
+import { sidecar, tamarind, juniper, dustStorm, whiteLilac } from '../constants';
 import WelcomeHeader from './welcomeHeader';
 
 const tableSize = 350;
```

Comment:
don't think we need this anymore, i think it was used to help with the calculations for where to place things

Reply:
yup I'll delete